filebeat/input/journald/pkg/journalfield/default.go

```diff
@@ -85,4 +85,5 @@ var journaldEventFields = FieldConversion{
 	"_SOURCE_REALTIME_TIMESTAMP":  ignoredField, // saved in the registry
```

Comment:
Is the comment on L84 inaccurate? It also says it is used for the timestamp.

Reply:
It seems like it is not accurate, as far as I understand looking at the current code. @belimawr am I missing some place where it is used?

Reply:
The comment on L84 was there before we started using [journalctl to read the journal](https://github.com/elastic/beats/pull/40061). Looking at the [old code](https://github.com/elastic/beats/blob/76f45fe41cbd4436fba79c36be495d2e1af08243/filebeat/input/journald/pkg/journalread/reader.go#L158-L191), the timestamp field was coming from the library we used, [here is the code](https://github.com/coreos/go-systemd/blob/d5623bf85e8e73ae6352f78ee6b55a287619dd4e/sdjournal/journal.go#L722-L760) that reads the journal and sets the timestamps.

When I re-wrote it to use `journalctl`, I looked at the code and `__REALTIME_TIMESTAMP` seemed the best choice to keep the behaviour of the old code/library.

So the comment there  might be misleading :/.

I did [write a test](https://github.com/elastic/beats/blob/372b1267b8802a4682d18af3cc915968ff373205/filebeat/input/journald/input_test.go#L153-L225) that compares both implementations reading the same journal file, so I believe the behaviour has not changed with the new implementation.